### ISSUE TITLE
iFrame with copy of parallel DOM

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#173]
It would be helpful to have a a document with the simulation in an iFrame with either the parallel DOM or a copy styled near it.  We could then see the parallel DOM and its content change during interaction.  This would allow development without searching for DOM elements in the insepector, which could save loads of time.  Also useful for testing without a screen reader, it will be easier to catch bugs without the overhead of enabling some AT.  Could also be useful for presentations and th like for showing a visualization of the parallel DOM concept.

Since I expect this to enhance development velocity, (and I don't expect it to be very trickty) marking as high accessibility priority.

**Comment by jessegreenberg** [comment]
> with the simulation in an iFrame with either the parallel DOM or a copy styled near it.

Must be a copy, if we remove the PDOM from iFrame inner document, the simulation cannot reference it.

**Comment by jessegreenberg** [comment]
How can we listen for changes to the PDOM, and copy to the document containing the iframe?

**Comment by jessegreenberg** [comment]
MutationObserver would probably work.  We need to listen for changes to the iframe document, which we can grab with something like 
```js
      var innerDoc = iframe.contentDocument || iframe.contentWindow.document;
```

**Comment by jessegreenberg** [comment]
This is *highly* inefficient, but for now, we can observe changes to the parallel DOM, and add an observer that makes a new copy of the PDOM, removes the old copy, and appends a new child to the document on every change.

**Comment by jessegreenberg** [comment]
The document extends below the cut off below:
![capture](https://cloud.githubusercontent.com/assets/6396244/19451118/4cf3cb5e-9469-11e6-83df-36025a6ec3e2.PNG)

Not perfect, but this should help quite a bit.  Looks like some inner styling isn't quite copied correctly. Closing for now.  Additional modifications can be discussed in new issues.
